source/infrastructure/dev-tools/index.html.md.erb

```diff
@@ -9,6 +9,9 @@ weight: 10
 
 Installation of each product should be carried out following the distributors current guidance, depending on your operating system.
 
+## Desktop Docker
```

Comment:
I believe it's commonly named "Docker Desktop" as per https://www.docker.com/products/docker-desktop/